src/global_logic.js

```diff
@@ -797,9 +808,9 @@ module.exports.calcBasedOneSummon = function (summonind, prof, buff, totals) {
         if (expectedTurn === Infinity) {
             expectedCycleDamagePerTurn = expectedAttack * damage;
 
-            ['contentious', 'supplemental_thirdHit'].some(key => function(key) {
-                if (supplementalDamageArray.includes(key)) {
-                    expectedCycleDamagePerTurn += supplementalDamageArray[key].damage * expectedTurn;
+            supplementalThirdHitNames.forEach(function(key) {
+            if (supplementalDamageArray[key] != undefined) {
```

Comment:
indent+4  and  forEach((key) => { ... }) style function for Array methods

Reply:
_removed._